app/components/Wallet/useWallet.tsx

```diff
@@ -93,5 +118,100 @@ export const useWallet = () => {
 		if (response.status === "success") setNetwork(newNetwork);
 	}, []);
 
-	return { isConnected, balance, network, connectWallet, disconnectWallet, switchNetwork };
+	const fetchUTXOs = useCallback(async (address: string): Promise<UtxoI[]> => {
+		try {
+			const response = await axios.get(`${MEMPOOL_API}/address/${address}/utxo`);
```

Comment:
I think wallet should provide that information, could you double check?

Reply:
Done